### PR TITLE
minutes from last friday

### PR DESCRIPTION
**Pull request by bumblefudge**

### CODE REVIEW

File: 2024-10-04/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+1. Intros
+2. Community announcements
+   * Darius: Open Standards advocate inside the Harvard Applied Social Media
+     Lab. Applied Social Media lab is interested in hosting convenings related
+     to decentralized social media / SWICG kind of things. Also will be
+     sponsoring Darius's work on the Social Media Schema Observatory.
+   * [IndieWeb Berlin!](https://indieweb.org/2024/Berlin) has organized and
+     opened up tickets for the event. Tickets are complimentary, two day barcamp
+     stype unconference and hackathon for all things independent and social web.
```
Comment (TallTed) [inline 2024-10-04/README.md:59]:
```suggestion
     type unconference and hackathon for all things independent and social web.
```

File: 2024-10-04/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+     Excited to be in Germany which is a hotbed of social web activity, will be
+     hosted at Mozilla Berlin.
+   * Social Web Foundation: Evan is now working with SWF with the goal of
+     improving the network and also the user experience. Not yet a W3C member
+     but will be soon, for now Evan is participating as individual.
```
Comment (TallTed) [inline 2024-10-04/README.md:64]:
```suggestion
     but will be soon; for now, Evan is participating as individual.
```

File: 2024-10-04/README.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        generally. I would like to make this explicit rather than implicit for
+        both the CG and the WG via elections. W3C has now set up a voting
+        mechanism for members and held elections for one WG so at least there is
+        precedent. For editors maybe a similar voting mechanism?
+      * Emelia: on the voting procedure, I was involved in the Solid one. There
+        was a comment from someone on the forums who was unhappy with how the
+        Solid election went, there was a perception that Inrupt carried lots of
+        weight and won a lot of board seats. Virginia ran as an independent
+        without Inrupt supporting her so it's important to note that while she
+        is Inrupt affiliated, she did not represent inrupt. We need to be clear
```
Comment (TallTed) [inline 2024-10-04/README.md:104]: Her affiliation has changed. I don't know the exact dates, but I'm pretty sure it happened before this meeting, and it _may_ have happened before that election. 
```suggestion
        was Inrupt affiliated, she did not represent Inrupt. We need to be clear
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Typos, punctuation, etc.

**Review by dmitrizagidulin — APPROVED** [review]

**Pull request merged**